lib/chai/utils/addChainableMethod.js

```diff
@@ -79,7 +78,7 @@ module.exports = function (ctx, name, method, chainingBehavior) {
 
         var assert = function assert() {
           var old_ssfi = flag(this, 'ssfi');
-          if (old_ssfi && config.includeStack === false)
+          if (old_ssfi)
```

Comment:
Could you please explain why you've removed the `config.includeStack` part here?